libs/application/templates/official-journal-of-iceland/src/screens/InvolvedPartyScreen.tsx

```
@@ -8,17 +8,24 @@ import { AlertMessage, Box, Stack } from '@island.is/island-ui/core'
   setSubmitButtonDisabled,
+  refetch,
 }: OJOIFieldBaseProps) => {
-  const { updateApplication } = useApplication({
+  const { updateApplication, submitApplication } = useApplication({
     applicationId: application.id,
   })
   const { formatMessage: f } = useLocale()
   const { setValue } = useFormContext()
 
+  useEffect(() => {
+    setSubmitButtonDisabled && setSubmitButtonDisabled(true)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)